packages/hash/frontend/src/blocks/page/BlockContextMenu/BlockContextMenuItem.tsx

```diff
@@ -65,6 +65,14 @@ export const BlockContextMenuItem = forwardRef<
           })}
       sx={{
         position: "relative",
+
+        [`&.${menuItemClasses.focusVisible}`]: {
+          boxShadow: "none",
+        },
+
+        ...(subMenuPopupState.isOpen && {
+          backgroundColor: ({ palette }) => `${palette.gray[20]} !important`,
+        }),
```

Comment:
```suggestion

        [`&.${menuItemClasses.focusVisible}`]: {
          backgroundColor: ({ palette }) => `${palette.gray[20]}`,
        },
        [`&.${menuItemClasses.root}`]: {
          boxShadow: "none",
        },
```

Reply:
We cannot use `focusVisible` classname  instead of `subMenuPopupState .isOpen` because the issue here is, focus stays there even if it's not hovered or menu is not open (because of MUI not supporting nested menu items, and our workaround). 

When the suggestion here is applied, it looks like this (wrong `backgroundColor` visible even if not hovered)
![Screen Shot 2022-09-05 at 12 35 35](https://user-images.githubusercontent.com/39553853/188418539-b1ed5861-9ad0-44c3-bd15-5b413ffabcbf.png)

Reply:
Would this work better?

```suggestion

        boxShadow: "none !important",

        "&:active": {
          backgroundColor: ({ palette }) =>
            `${palette.primary.main} !important`,
        },
        
        ...(subMenuPopupState.isOpen && {
          backgroundColor: ({ palette }) => `${palette.gray[20]} !important`,
        }),
```

Reply:
Yeah, thanks for noticing the `&:active` issue, it should be fixed here https://github.com/hashintel/hash/pull/1020/commits/bd6ac3803f64d09e87d615f255c2d6d40f63ee71